service/src/main/java/uk/gov/hmcts/reform/fpl/service/additionalapplications/UploadAdditionalApplicationsService.java

```diff
@@ -105,6 +115,28 @@ public AdditionalApplicationsBundle buildAdditionalApplicationsBundle(CaseData c
         return additionalApplicationsBundleBuilder.build();
     }
 
+    private void buildC2BundleByPolicy(CaseData caseData, C2DocumentBundle c2Bundle,
+                                      AdditionalApplicationsBundleBuilder builder) {
+        final Set<CaseRole> caseRoles = userService.getCaseRoles(caseData.getId());
+
+        final Consumer<String> setConfidentialC2 = (fieldName) -> {
```

Comment:
parentheses around field name is not needed.

Reply:
updated